src/App.vue

```diff
@@ -26,6 +26,11 @@ export default {
     isSmallDevice() {
       return this.$vuetify.display.smAndDown;
     },
+    theme() {
+      const isHomeView = this.$router.currentRoute.value.name === 'Home';
+      const theme = isHomeView ? 'light' : 'dark'; 
+      return theme;
```

Comment:
No need for the `theme` variable if we're just returning it
```suggestion
      return isHomeView ? 'light' : 'dark'; 
```